odoo/fields.py

```diff
@@ -3424,7 +3424,8 @@ def _setup_regular_base(self, model):
             )
         if self.store:
             if not (self.relation and self.column1 and self.column2):
-                self._explicit = False
+                if not self.relation:
+                    self._explicit = False
```

Comment:
@rco-odoo
it's needed to add field product_template_variant_value_ids without explicit column names